extensions/telemetry/package.json

```diff
@@ -9,8 +9,8 @@
     "styles": []
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js",
-    "dev": "npm run build --watch"
```

Comment:
For some reasons reusing command didn't work for me from WebStorm (maybe because need of `$@` at the end or different running context, dunno). Btw `webpack.config.js` is a default config name (even `.ts` is supported, when used in conjunction with `ts-node`).